datafusion/common/src/config.rs

```diff
@@ -901,6 +941,8 @@ pub trait ConfigExtension: ExtensionOptions {
 }
 
 /// An object-safe API for storing arbitrary configuration
+/// do
```

Comment:
This doesn't seem to be displayed very well in cargo doc.
![CleanShot 2025-01-01 at 23 44 10@2x](https://github.com/user-attachments/assets/9cc2197a-d1f1-479e-b542-b348ce87f939)

Reply:
Thank you for this good catch. I am not sure why I made this change. Fixed in c433575f8

It looks much better now 

![Screenshot 2025-01-01 at 12 44 19 PM](https://github.com/user-attachments/assets/3686d8c2-5e22-4855-bcab-d689d23ab7a8)